### DISCUSSION TITLE
CUSA13760 - Conception Plus Maidens of The Twelve Stars

### DISCUSSION THREAD

**Discussion by SrPhreaK** [GoldHEN/GoldHEN_Cheat_Repository#29] [Cheat requests]
when possible ,
I would be happy to see cheats for this game.

thanks to everyone who has already helped the ps4 scene,
it certainly makes a difference to many in the world.

**Comment by SrPhreaK** [comment]
it doesn't have to be anything absurd.
it may just be money related.

but if you have something related to experience , or items , even better .

**Reply by SrPhreaK** [reply]
@kmeps4 Could you please take a look when possible? Thanks

**Comment by kmeps4** [comment]
https://github.com/GoldHEN/GoldHEN_Cheat_Repository/discussions/42#discussion-4222094
